src/settings/mod.rs

```diff
@@ -37,7 +25,7 @@ impl Settings
   ///
   /// Precedence (earlier items override later ones):
   ///
-  ///   1. Environment variables: `$FXA_<UPPERCASE_KEY_NAME>`
+  ///   1. Environment variables: `$FXA_EMAIL_<UPPERCASE_KEY_NAME>`
```

Comment:
I opted to make the env var prefix a bit more specific, to avoid annoying future us if we have this service running alongside all the others in local dev.